styleshet.css

```diff
@@ -77,7 +77,11 @@ td:after {
 }
 
 td .content:active {
-    transform: scale(0.9);
+    transform: scale(1.05);
```

Comment:
If you click on the edge of an element, transforming it to a smaller size prevents the click event from being fired